### PR TITLE
Add script to sync bundle

### PR DESCRIPTION
**Pull request by gnufied**
Add commands to generate csv and bundle and index image

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -16,7 +16,31 @@ and then update `deploy/operator.yaml` to point to your images and follow rest o
...
+~> ./hack/sync_bundle -o <operator_image> -d <diskmaker_image> -b <bundle_image> -i <index_image> bundle
+
+~> ./hack/sync_bundle -o quay.io/openshift/origin-local-storage-operator:latest  \
+        -d quay.io/openshift/origin-local-storage-diskmaker:latest \
+        -b quay.io/gnufied/local-storage-bundle:v1 \
+        -i quay.io/gnufied/gnufied-index:v1 bundle
+~> docker push quay.io/gnufied/gnufied-index:v1
+```
+
+This should give us index image `quay.io/gnufied/gnufied-index:v1` which we can use in `examples/olm/catalog-create-subscribe.yaml` file:
```
Comment (huffmanca) [inline HACKING.md:35]: Personal nitpick - I'd remind that the `examples/olm/catalog-create-subscribe.yaml` file needs to be updated before we create it to use the index image. So maybe:

> This should give us index image `quay.io/gnufied/gnufied-index:v1`.
> Update the `CatalogSource` entry in `examples/olm/catalog-create-subscribe.yaml` to point to your newly created index image. Once updated, we can create this without further changes:
> ```
> ~> oc create -f examples/olm/catalog-create-subscribe.yaml
> ```

Comment (gnufied) [reply HACKING.md]: done

**Pull request merged**